components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -398,6 +647,15 @@ public void testChownAnnotationReadAnnotate(boolean isDataOwner, boolean isAdmin
                         testCase[IS_GROUP_OWNER] = isGroupOwner;
                         testCase[IS_RECIPIENT_IN_GROUP] = isRecipientInGroup;
                         testCase[IS_EXPECT_SUCCESS] = isAdmin || isGroupOwner && isRecipientInGroup;
+                        for (final Option value : values) {
+                            /* only add one tag set child option per one permission
+                             * test, but alternate over all child options doing this */
+                            if (count_value < value.ordinal()) {
+                                continue;
+                            }
+                            testCase[CHILD_OPTION] = value;
+                        }
+                        count_value = (count_value + 1) % values.length;;
                         // DEBUG: if (isDataOwner == true && isAdmin == true && isGroupOwner == true &&
                         //            isRecipientInGroup == true)
```

Comment:
add option in here too: the idea of these is that you uncomment them and suddenly it picks out just the test case you want instead of the combination of them all